.github/workflows/ci.yml

```diff
@@ -93,7 +82,7 @@ jobs:
         env:
           TEMPORAL_TEST_PROTO3: 1
         run: |
-          poetry add "protobuf<4"
+          poetry add --python 3.9 "protobuf<4"
```

Comment:
Worth a comment explaining the `--python 3.9` here?

Reply:
I admit I do not remember exactly, but I _think_ it was saying there wasn't compatibility with some of the other dependencies